pkg/imager/fyne_helpers.go

```diff
@@ -70,8 +71,21 @@ func showErr(fg *FyneUI, err ...error) bool {
 	return false
 }
 
+func insertNth(s string, n int) string {
+	var buffer bytes.Buffer
+	var n_1 = n - 1
+	var l_1 = len(s) - 1
+	for i, rune := range s {
+		buffer.WriteRune(rune)
+		if i%n == n_1 && i != l_1 {
```

Comment:
If someone provides a string that contains newlines, this will introduce unnecessary newlines. I think at least we should check for the distance from the last newline character.

Also, it will insert newline as exactly 80th character, possibly breaking a word in two. It might be better to split on whitespaces whenever it's possible. However, it's not trivial and probably an overkill in this case.